include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -268,6 +270,17 @@ typedef struct OrtCUDAProviderOptions {
   int do_copy_in_default_stream;
 } OrtCUDAProviderOptions;
 
+/// <summary>
+/// Options for the ROCM provider that are passed to SessionOptionsAppendExecutionProvider_ROCM
+/// </summary>
+typedef struct OrtROCMProviderOptions {
+  int device_id;                                    // hip device with id=0 as default device.
+  OrtMIOpenConvAlgoSearch miopen_conv_algo_search;  // miopen conv algo search option
```

Comment:
This option was added for CUDNN. Not sure whether this option is applicable to miopen or not. If not, let's delete it.

Reply:
changed it to a bool data type.